source/fundamentals/crud/run-command.txt

```diff
@@ -0,0 +1,167 @@
+.. _golang-run-command:
+
+=============
+Run a Command
+=============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to run a **database command** with the
+{+driver-short+}. You can use database commands to perform a variety of
+administrative and diagnostic tasks, such as fetching server statistics,
+initializing a replica set, or running an aggregation pipeline.
+
+Compose a Command
+-----------------
+
+The {+driver-short+} provides the following methods to run database
+commands: 
+
+- ``RunCommand()``, which returns the command response as a
+  ``SingleResult`` type. You can use this method with any database command.
+- ``RunCommandCursor()``, which returns the command response as a
+  ``Cursor`` type. You can use this method if your database command
+  returns multiple result documents.
+
+Each method requires a Context and a command document. The
+command document must be an order-preserving type such as ``bson.D``.
```

Comment:
Issue: command document hasn't been defined, so it can be confusing to the reader to only read about the type requirements rather than what it is. I think it could be helpful to explain the parts of the command document since it isn't self-evident in the example (e.g. what's the "1" value used for?)